AWS/AWS_-_Identify_Unused_VPCs.yaml

```diff
@@ -0,0 +1,144 @@
+- defaultTab: output
+  description: |
+    Generate list of unused Virtual Private Networks
+
+    ### Description:
+    The automation job is designed to identify VPCs that are unused across mulitiple resource types.  They would be candidates for removal.
+
+    ### Prerequisites:
+    - AWS-CLI installed on runner.
+    - API permissions for
```

Comment:
need permissions here

Reply:
Pushed a new file generated based on the other doc pages